dts/bindings/audio/nxp,dmic.yaml

```diff
@@ -32,12 +32,12 @@ child-binding:
   description: |
     NXP DMIC channel. Can be used to configure filtering and gain attributes
     of each channel
-  include: base.yaml
-  compatible: "nxp,dmic-channel"
   properties:
     reg:
+      type: array
```

Comment:
This can be deleted. We want to reuse the property from `base.yaml` and make this property required. Same comment for the below change.

Reply:
but that's now how child-bindings work, AFAICT. There are other bindings in-tree that use the same approach, and AFAICS no single child-binding is including base.yaml.

FWIW the `nxp,dmic-channel` compatible effectively does *not* exist (https://docs.zephyrproject.org/latest/build/dts/api/bindings.html#dt-vendor-nxp) so, again, while it's certainly confusing, I just think the child-binding should (re)define whatever property it needs